erpnext/domains/vehicles.py

```diff
@@ -332,6 +332,17 @@ def get_field(fieldname, field_list):
 		"insert_after": "designation"},
 ]
 
+#Insurace Details
```

Comment:
Remove all custom fields, add fields by modifying DocType instead of custom fields because all the insurance related fields are already in DocType